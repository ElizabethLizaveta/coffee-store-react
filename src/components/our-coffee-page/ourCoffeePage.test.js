import React from 'react'; 
import OurCoffeePage from './ourCofeePage';
import {shallow} from 'enzyme';


describe('Testing <OurCoffeePage/>', () => {
it('FilterPanel have rendered correctly', () => {
    const char = shallow(<OurCoffeePage/>);
    expect(char).toMatchSnapshot();
})
});