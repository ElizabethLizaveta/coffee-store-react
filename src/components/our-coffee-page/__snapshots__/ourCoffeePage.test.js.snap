// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <OurCoffeePage/> FilterPanel have rendered correctly 1`] = `ShallowWrapper {}`;
